ci/cron/src/Main.hs

```diff
@@ -176,6 +176,15 @@ build_docs_folder path versions current = do
         let (releases, snapshots) = List.partition (not . prerelease . fst) documented_versions
         create_versions_json (map fst releases) (new </> "versions.json")
         create_versions_json (map fst snapshots) (new </> "snapshots.json")
+        -- Starting after 0.13.54, we have changed the way in which we trigger
```

Comment:
CI is failing because `latest_sha` is now unusued.

Reply:
🤦